### ISSUE TITLE
Cannot select app to share note with

### ISSUE THREAD

**Issue by user6529576559** [standardnotes/android#107]
First time it asked which app I want the note to share with, I selected it, shared the note, everything was OK. Next time I wanted to share an other note, then I saw that the app selection step is skipped and it wants to share with the previously selected app.

**Comment by deftelf** [comment]
What version of Android are you using please?

**Comment by user6529576559** [comment]
Well, this info will hurt my pseudonimity, but 6.0.1

**Comment by deftelf** [comment]
I don't think this an app bug - we're doing nothing special. Could you have clicked "always" share instead of "just once"? Could you try clear the defaults (see bottom of this thread) and try again?

http://forums.androidcentral.com/android-4-4-kitkat/332563-share-menu-won-t-open-up-anymore-only-shares-one-app.html

**Comment by user6529576559** [comment]
There is no options like "always" or "just once" in case of sharing. I tried the suggestion you linked, it solved the problem but only for once. So I should reset the settings after every share or when I want to share with a different app. Other apps doesn't have this problem. Very strange, maybe a bug in android, but why only for standard notes?

**Comment by deftelf** [comment]
Hm. Could you reset it again, and attach a screen shot of the popup? Also, what device are you using?

**Comment by user6529576559** [comment]
The strange thing is that the problem does not appear when I use the Export Data button in the settings screen. I'll provide a screenshot later.

**Comment by user6529576559** [comment]
![ss](https://cloud.githubusercontent.com/assets/25223090/25897702/65b6b06a-3578-11e7-99d0-e915a76ba341.png)
Samsung Galaxy S5 Mini

**Comment by user6529576559** [comment]
I think maybe the problem is that the "open with" dialog is shown instead of the "share with" dialog. (Based on the "Complete action using" text.)

**Comment by deftelf** [comment]
f014dbc7e3ba0e28b3e9840cd1458081c96b1040

**Comment by deftelf** [comment]
I have made an innocuous change for v1.3 that might fix this. Please let me know!

**Comment by user6529576559** [comment]
I tried it in 1.3 (beta) but this bug is still there.

**Comment by deftelf** [comment]
We really could do with a developer with a Samsung phone to look into this if anyone is available
